storage/innobase/include/dict0mem.h

```diff
@@ -1537,10 +1537,18 @@ struct dict_table_t {
 	/** Add the table definition to the data dictionary cache */
 	void add_to_cache();
 
-	bool versioned() const { return vers_start || vers_end; }
+	/** Detects if the table is versioned.
+	It is assumed that both vers_start and vers_end set to 0
+	iff table is not versioned. In any other case this fields correspond to
+	actual column indexes.
+	 */
+	bool versioned() const
+	{
+		return vers_start || vers_end;
+	}
```

Comment:
I would keep the code in a single line, so that `git grep versioned` will show the function definition.
In the comment, please do not overload the word ‘index’, because normally there is no index on the `vers_end` column. Say ‘position in `cols[]`’ instead. (‘Column position’ alone could be somewhat ambiguous, potentially referring to clustered index field position.)

Start the comment with `@return`, and move the `*/` to the end of the last non-whitespace line. Fix the grammar: ‘this fields’ should be ‘these fields’.

Reply:
updated. check if everything is good now

Reply:
oh no, I can't push to tempesta anymore, so the pull request could not be updated now. please check the https://github.com/MariaDB/server/commits/bb-10.3-trxid_fixes
I will close the PR and push the commits to 10.3 when you'll resolve the conversation